docker/ml/Dockerfile

```
@@ -12,7 +12,16 @@ RUN apt-get update \
 COPY demisto_ml-0.2.tar.gz .
 RUN pip install demisto_ml-0.2.tar.gz
 RUN rm -rf /var/lib/apt/lists/*
+
+WORKDIR /ml
+RUN useradd -m ml
+RUN chown -R ml:ml /ml
+
+USER ml
+
 RUN python -m spacy download en_core_web_sm
-RUN python -c "import nltk; nltk.download('stopwords'); nltk.download('punkt')"
-COPY encrypted_model.b ./var
-COPY oob_evaluation.txt ./var
+RUN python -c "import nltk; nltk.download('stopwords', download_dir='/ml/nltk_data'); nltk.download('punkt',download_dir='/ml/nltk_data')"
+ENV NLTK_DATA='/ml/nltk_data'
+ENV MPLCONFIGDIR='ml/matplotlib'
```

Don't you need a `/`
```suggestion
ENV MPLCONFIGDIR='/ml/matplotlib'
```

---

thanks